partcad/src/partcad/part_factory_brep.py

```
@@ -0,0 +1,134 @@
+            response_serialized, errors = await self.runtime.run_async(
+                [
+                    wrapper_path,
+                    os.path.abspath(self.path),
+                    os.path.abspath(self.project.config_dir),
+                ],
+                request_serialized,
+            )
+            if errors:
+                sys.stderr.write(errors)
+
+            response = pickle.loads(base64.b64decode(response_serialized))
```

Ruff (0.8.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)